variables.tf

```diff
@@ -41,25 +41,13 @@ variable "reports_schedule" {
 }
 
 variable "scan_schedule" {
-  description = "schedule for running domain-protect scans, e.g. 60 minutes, does not affect frequency of regular Slack reports"
-  default     = "60 minutes"
-  type        = string
-}
-
-variable "scan_schedule_nonprod" {
-  description = "schedule for running domain-protect scans in non-prod, reduced to save costs, e.g. 12 hours"
+  description = "schedule for running domain-protect scans, e.g. 24 hours"
   default     = "24 hours"
```

Comment:
Instead of defaulting it to all the `nonprod` values. Shouldn't we default it to all the `prod` values since those are already the existing defaults ?

Reply:
https://owasp.slack.com/archives/C04BPJ5B2P4/p1724159709558629?thread_ts=1724097809.543719&channel=C04BPJ5B2P4&message_ts=1724159709.558629

Reply:
Make sense. Thanks